token-lending/program/src/instruction.rs

```diff
@@ -30,6 +30,18 @@ pub enum LendingInstruction {
         owner: Pubkey,
```

Comment:
This commit makes no changes except to reorder the instructions and processor functions.